dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -95,11 +95,11 @@ def new_general_foorm
         sessionId: session&.id,
       }
 
-      if Pd::WorkshopSurveyFoormSubmission.has_submitted_form?(current_user.id, workshop.id, session&.id, day, survey_name)
+      if !params[:force_show] && Pd::WorkshopSurveyFoormSubmission.has_submitted_form?(current_user.id, workshop.id, session&.id, day, survey_name)
         return redirect_general(key_params)
       end
 
-      form_questions, latest_version = Foorm::Form.get_questions_and_latest_version_for_name(survey_name)
+      form_questions, latest_version = ::Foorm::Form.get_questions_and_latest_version_for_name(survey_name)
```

Comment:
Do we maybe need an `include` to avoid the new `::`?

Reply:
doesn't seem to help. I think it's needed because I added Pd::Foorm module so now just using Foorm wants to try to find under Pd::Foorm. I could take the Pd out of the new module and just name it Foorm to keep consistency though